### DISCUSSION TITLE
transparent background

### DISCUSSION THREAD

**Discussion by Wateir** [adriankarlen/textfox#63] [Q&A]
![image](https://github.com/user-attachments/assets/1ea185ae-2d0f-4257-9237-edcecfbfbe96)

I want a complete transparent bacground, i have currently the main and tab margin transparent, but i can have the navbar margin to be transparent.

For achive the look i have currently, i change on userChrome.css for 
`body {
  background-color: color-mix(in srgb, currentColor 10%, transparent);
}
`

**Comment by adriankarlen** [comment]
The `#navigator-toolbox` has it's own background.

if you add `background-color: transparent` to it, you should have your desired outcome.

**Comment by Wateir** [comment]
`background-color: transparent` has no change

`background-color: transparent !important` make a weird look

![image](https://github.com/user-attachments/assets/93f044ca-2c2a-475c-bc50-4865efe96682)

as you can see, it's only the left part who is now transparent.

(gonna add a background color after on the right place)

**Comment by adriankarlen** [comment]
Okay, I'd suggest inspecting with the Browser Toolbox to find out what is setting what background.

**Comment by Wateir** [comment]
gonna try this, sorry for bother you, i'm really not good with this hight level langagues, specialy on ui jobs

**Comment by Wateir** [comment]
![image](https://github.com/user-attachments/assets/ef04efbb-e76a-4098-ac3b-e54fe1b6c561)

Okay so far i'm at this point, i have basicly all i want, but the transparency have some issue 

`#navigator-toolbox {
  background-color: transparent !important;
}
#nav-bar-customization-target{
	background-color: #000000 !important; 
}
#nav-bar {
  background-color: transparent !important;
  }
`

i not really sure where come the issue

but it's something responsive

![image](https://github.com/user-attachments/assets/d47da31c-e16a-418f-9cb1-2c6fc4b0c3d3)

![image](https://github.com/user-attachments/assets/ba82f0c2-3c49-4f98-a1cc-68e17b228836)
 
the lenght don't change but the part transparent change, can find the issue with the browser toolbox (btw thanks for the tips)

if i find a solution, gonna send it

**Reply by HeyUKonrad** [reply]
did you ever find a solution?
